lightkurve/tests/test_targetpixelfile.py

```diff
@@ -51,6 +55,20 @@ def test_tpf_plot():
     with pytest.raises(ValueError):
         tpf.plot(scale="blabla")
 
+def test_find():
```

Comment:
Please add `@pytest.mark.remote_data` above `def test_find():` -- it tells the testing system to skip this test if no internet is available.